src/main.js

```diff
@@ -958,6 +990,7 @@ function loadSettings() {
  */
 function createWebView(id, { url, zoom, customCSS, forOverlay, forSmallPane }) {
   const webview = new WebView({ url, zoom, customCSS });
+  webview.addShortcutKey("meta+l", webview => openUrlChangeDialog(webview));
```

Comment:
`this`コンテキスト使って無ければこれで済むかも…？

```suggestion
  webview.addShortcutKey("meta+l", openUrlChangeDialog);
```

Reply:
済んじゃった。いぇーい。
関数名だけを指定したときにコールバックの引数に何が渡されるのか理屈を全く理解しておらぬ…